tests/common.go

```diff
@@ -348,7 +348,7 @@ func createK8sClientWithConfig(t *testing.T, restCfg *rest.Config) kubernetes.In
 		return retryClient.StandardClient().Transport
 	}
 
-	restCfg.Timeout = 8 * time.Second
+	restCfg.Timeout = 10 * time.Second
```

Comment:
A comment about the conditions (>context timeout?) for this timeout would be helpful.
Could it be reasonably computed from the context timeout?

Reply:
but context does not exposes timeout, maybe we can get it with reflection but for now I think it'll be fine to increase it